corgisim/outputs.py

```
@@ -389,69 +398,101 @@ def write_headers_CFAM(band_pass):
     
     
 
-def write_headers_DPAM(cor_mode, polaxis):
+def write_headers_DPAM(cor_mode, polaxis, prism):
      ### determine the value for DPAM based on simulation mode and polaxis number
     
     if (cor_mode == 'excam') & (polaxis in ['0', '10']):
+        ## excam imaging mode, no polarimetry
         DPAM_H = 38917.1
         DPAM_V = 26016.9
         DPAMNAME = 'IMAGING'
         DPAMSP_H = 38917.1
         DPAMSP_V = 26016.9
     if cor_mode == 'spec':
-        raise ValueError('Spec mode has not been implemented')
+        #raise ValueError('Spec mode has not been implemented')
+        if prism == 'PRISM2':
+            DPAM_H = 62496
+            DPAM_V = 1261.3
+            DPAMNAME = 'PRISM2'
+            DPAMSP_H = 62496
+            DPAMSP_V = 1261.3
+        if prism == 'PRISM3':
+            DPAM_H = 26824.2
+            DPAM_V = 1261.3
+            DPAMNAME = 'PRISM3'
+            DPAMSP_H = 26824.2
+            DPAMSP_V = 1261.3
+        if prism == 'None':
+            DPAM_H = 38917.1
+            DPAM_V = 26016.9
+            DPAMNAME = 'IMAGING'
+            DPAMSP_H = 38917.1
+            DPAMSP_V = 26016.9
```

should we throw an error if a prism that isn't support get input?